modules/site/src/app/examples/input/input-basic-usage.component.ts

```diff
@@ -22,15 +22,26 @@ export class InputBasicUsageComponent {
     address2: '',
     city: 'Mountain View',
     state: 'CA',
-    biography:
-        'Loves kittens, snowboarding, and can type at 130 WPM.\n\nAnd rumor has it she bouldered up Castle Craig!',
+    biography: 'Loves kittens, snowboarding, and can type at 130 WPM.\n\nAnd rumor has it she bouldered up Castle Craig!',
     postalCode: '94043'
   };
 
+  form:FormGroup;
+  postalCode:FormControl;
+
   states = [
     'AL', 'AK', 'AZ', 'AR', 'CA', 'CO', 'CT', 'DE', 'FL', 'GA', 'HI', 'ID', 'IL',
     'IN', 'IA', 'KS', 'KY', 'LA', 'ME', 'MD', 'MA', 'MI', 'MN', 'MS', 'MO', 'MT',
     'NE', 'NV', 'NH', 'NJ', 'NM', 'NY', 'NC', 'ND', 'OH', 'OK', 'OR', 'PA', 'RI',
     'SC', 'SD', 'TN', 'TX', 'UT', 'VT', 'VA', 'WA', 'WV', 'WI', 'WY'
-  ].map(function(state) { return {abbrev: state}; })
+  ].map(function (state) {
```

Comment:
I didn't touch that part. My IDE just did some code reformat that's why the line have changed.

As this is unused code, I propose to comment it for now.